core/src/test/java/org/apache/struts2/dispatcher/DispatcherTest.java

```diff
@@ -355,6 +355,10 @@ public void testIsMultipartRequest() throws Exception {
 
         req.setContentType("Multipart/Form-Data ; boundary=---------------------------207103069210263;charset=UTF-16LE");
         assertTrue(du.isMultipartRequest(req));
+
+        //test order-insensitive, tabs and tspecials characters
```

Comment:
tspecials -> special

Reply:
I gave it from the reference (https://tools.ietf.org/html/rfc2045#section-5.1) - maybe it means token specials.